tests/multigrid/test_netgen_gmg.py

```diff
@@ -0,0 +1,73 @@
+from firedrake import *
+import pytest
+
+
+@pytest.mark.skipcomplex
+@pytest.mark.skipnetgen
+def test_netgen_mg_circle():
+    from netgen.geom2d import Circle, CSG2d
+    geo = CSG2d()
```

Comment:
please replace the duplicated code with a function that you call from both tests.